src/Utility/Discord/ReactionCollector.js

```diff
@@ -0,0 +1,173 @@
+import { EventEmitter } from 'events';
+
+import { Collection } from '../Collection';

```

Comment:
This should just be:
```suggestion
import Collection from '../Collection';
```
This is what caused the error we were wondering about. 
However, the error was indeed confusing and I can see why you couldn't find what was the cause.

Reply:
Yes, that was exactly what was preventing the bot from running. Fixed that and is functioning now.